SUMMARY.md

```diff
@@ -217,9 +217,11 @@
   - [Service Portal Wip](libs/service-portal/wip/README.md)
 - [Shared](libs/shared/README.md)
   - [Babel](libs/shared/babel/README.md)
+  - [Shared Components](libs/shared/components/README.md)
```

Comment:
Shouldn't the item just be named 'Components' instead of 'Shared Components' as it nested under 'Shared'?

Reply:
Good point. Will fix.

Reply:
Though this uses the title of the library README. Not sure if "Components" is a good title for the library. What do you think @jeremybarbet?

Reply:
We used `UI` prefix for the application system https://github.com/island-is/island.is/blame/main/SUMMARY.md#L156-L158 and I like it.

Regarding the summary itself, I tried to remove the folder name from the subfolder title, otherwise, it's a bit repetitive but I can see a bunch of API Domains under "Domains" and that are using "API Domains ..." in the title and same goes with "Applications templates"

Reply:
Talked with Jeremy. Going to keep this like is and later implement a way to specify navigation title separate from the markdown title, so we can have a full heading "Shared Components" in the readme, but skip the folder name in the navigation ("Components").